ql/src/test/results/clientpositive/llap/orc_merge5.q.out

```diff
@@ -385,6 +385,7 @@ STAGE PLANS:
   Stage: Stage-0
     Concatenate
       table name: default.orc_merge5b_n0
+      table name: default.orc_merge5b_n0
```

Comment:
nit: there seems duplicate, the same outputs twice

Reply:
No, This is not a duplicate. Input and output might be outputted here. If it was a duplicate, we would have had an error in the output file.

Reply:
I wondering the two one is from AlterTableConcatenateDesc,
https://github.com/apache/hive/blob/7b3ecf617a6d46f48a3b6f77e0339fd4ad95a420/ql/src/java/org/apache/hadoop/hive/ql/ddl/table/storage/concatenate/AlterTableConcatenateDesc.java#L57-L60
the other comes from AbstractAlterTableDesc:
https://github.com/apache/hive/blob/7b3ecf617a6d46f48a3b6f77e0339fd4ad95a420/ql/src/java/org/apache/hadoop/hive/ql/ddl/table/AbstractAlterTableDesc.java#L64-L67

Reply:
Ack